config/manager/manager.yaml

```diff
@@ -28,6 +28,18 @@ spec:
         image: controller:latest
         imagePullPolicy: Always
         name: manager
+        ports:
+        - containerPort: 9440
```

Comment:
Is this a standard port for probes?

Reply:
I don't know if exists a standard port for probes, what I did was checking what was the `webhook-port` and round to the close 0.
`webhook-port` -> `9443`
`health-addr` -> `9440`

we can change to any other if you want 😄 please let me know

@ncdc thanks for your review and comments, PTAL.
question: do we want to do similar to the `cluster-api` as well? if yes I can work in that.

Reply:
Let's get this one finalized, then yes it would be great if you could do cluster-api too. There's also cluster-api-provider-{vsphere,azure,openstack,...} 😄

Reply:
From all the examples I've seen online, `8080` seems to be the most common healthz port https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/

Reply:
We do already have metrics defaulting to 8080, however.

Reply:
8081? :D

Reply:
I forget if the metrics are prometheus based, but we could potentially use one of the default prometheus ports instead of 8080

Reply:
I don't have a strong opinion on that, but most of the time that uses 8080 is because (maybe) it is using the same port as the webapp, and the webapp exposes a ping/health endpoint.
I think we can use anyone we choose.

for prometheus, i will open another PR, to change the port